coding-concepts.js

```diff
@@ -8,48 +8,47 @@
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
-// console.log(colors.push("indigo"))
-
-// a) Your answer:
-// b) Verify and explain:
+//console.log(colors.push("indigo"))
 
+// a) Your answer: will display indigo at the front spot of array
+// b) Verify and explain: It did push indigo into array but displayed the number of elements in the array
 // --------------------1) What will this log?
 
-const cohort = "LEARN 2023"
-// console.log(cohort.length)
+//const cohort = "LEARN 2023"
+ //console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: should log the length of the characters in the string which is 9
+// b) Verify and explain: so it logged 10 bc it is a string and not an array or else it would have been 0 indexed
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+//console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: should log the [4] placement in the string which would be l
+// b) Verify and explain: logged "o" which makes sense bc whenever calling on the index of a string it starts at 0
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
-// console.log(languages[index])
+//console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: i think it will log the ruby bc it is calling on the index placement [1]
+// b) Verify and explain: logged "ruby" bc arrays with stings are 0 indexed 
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
-// console.log(weekendDays.toUpperCase())
+ //console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: will call on strings in array and should return strings with all characters uppercased 
+// b) Verify and explain: so logged it and returned an error saying to uppercase is not a function. Maybe bc there was no specification of which element in the array.
 
 // --------------------5) What will this log?
 
 const dataTypes = ["number", "string", "Boolean", "undefined"]
-// console.log(typeof dataTypes.length)
+ //console.log(typeof dataTypes.length)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: i think this will return the amount of characters in each string in the array. So 5, 5, 6, and 8
+// b) Verify and explain: just logged the first element in the array. typeof is used for string so soon as it read that it stopped producing the function
```

Comment:
Be sure to research why you're not getting the output you think you should be.  I encourage you to go back and answer these again so you'll better understand the concepts.